helmfile/50-applications.yaml

```diff
@@ -453,6 +453,17 @@ releases:
   values:
     - values/falco-exporter.yaml.gotmpl
 
+# Blackbox in wc cluster
```

Comment:
maybe use the [Velero example](https://github.com/elastisys/compliantkubernetes-apps/blob/blackbox-wc/helmfile/50-applications.yaml#L51)
and keep one release for blackbox for both sc and wc

Reply:
i tried that multiple times but i didn't worked .

Reply:
When you tried to use the same setup as the velero example, did you also make sure to move the code block above [line 150](https://github.com/elastisys/compliantkubernetes-apps/blob/4340123bb8c4fcc9140081ee2fa655bb6f65f47a/helmfile/50-applications.yaml#L150)? Otherwise it would just be installed for sc (which is what I assumed happened for you).

Reply:
Thanks @viktor-f It worked now :) !! i will update the PR now :)

Reply:
Glad to hear :slightly_smiling_face: